### ISSUE TITLE
FD12FLOPPY.zip doesn't exist

### ISSUE THREAD

**Issue by patham9** [knazarov/homebrew-qemu-virgl#92]
Error: qemu-virgl: Failed to download resource "qemu-virgl--test-image"
Download failed: https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.2/FD12FLOPPY.zip

Is there any workaround?

**Comment by patham9** [comment]
I can confirm this is solved by https://github.com/knazarov/homebrew-qemu-virgl/pull/85
which just isn't yet merged into master. Of course feel free to close the issue once it is merged.

**Comment by ghost** [comment]
Please get this merged we need the fix for m1 as it is now we can not install

**Comment by esologic** [comment]
+1, Please merge. I need my acceleration :)

**Comment by 111andre111** [comment]
Just in case you didn't see, you can use this workaround that I confirm it's working: https://github.com/knazarov/homebrew-qemu-virgl/issues/83#issuecomment-1051147726

**Comment by sanathusk** [comment]
+1, Please merge

**Comment by sbates130272** [comment]
+1, please merge

**Comment by abrekhov** [comment]
+1 😄
